ios/build_and_install_errors.md

```diff
@@ -0,0 +1,41 @@
+# Build & Installation Errors
+
+When building your project locally, or remotely on a CI server, there are some common errors you may run in to. Similarly, installing ad-hoc and distribution apps from an OTA server can also pose some problems. Some errors can be fixed within your project configuration, or within the code itself -- however, some errors may require administrator privileges.
+
+## Build Errors
+
+##### Q. My build fails with "ssh: Could not resolve hostname"
+A. This is a general network error and indicates the network dropped at some point in the build. Restart the build to try it again.
+
+##### Q. I see a lot of "error: unable to read module map contents"
+A. This error happens when there was an issue installing Cocoapod dependencies. Ensure there weren't any odd changes to your Podfile committed. If you're referencing code that's been pushed since midnight of the last day, it's possible that the Cocoapods master-repo hasn't been updated yet, and doesn't contain spec information for the pod you're referencing.
```

Comment:
is this really possible though? in theory if this happens shouldnt we be getting cocoapods errors like 'could not resolve dependency..'?